ui/pages/swaps/dropdown-search-list/index.scss

```diff
@@ -126,7 +125,6 @@
 
     padding: 16px;
     color: var(--Grey-500);
-    min-height: 300px;
```

Comment:
This property is redundant and is being overridden by `min-height: 194px`, below.